Formula/couchdb.rb

```diff
@@ -78,13 +133,68 @@ def post_install
     end
   end
 
-  def caveats; <<-EOS.undent
+  def caveats
+    str = <<-EOS.undent
```

Comment:
Annoying nit but we tend to use a simple `s` rather than `str` for caveats like these across formulae.